Applications/Internet/Internet Explorer 6.0/Online/script.js

```diff
@@ -31,7 +31,7 @@ new PlainInstaller().withScript(() => {
 
     const wine = new Wine()
         .wizard(setupWizard)
-        .prefix("InternetExplorer6", "upstream", "x86", getLatestStableVersion(setupWizard, "x86"))
+        .prefix("InternetExplorer6", "upstream", "x86", getLatestStableVersion(setupWizard, null, null, "x86"))
```

Comment:
Did you try this? `PlainInstaller` is not a `QuickScript`, so I don't see how `this._wineDistribution` etc. should be set. Generally, I think it would be better to explicitly pass all parameters everywhere (`Versions` should provide a set of functions and not depend on `QuickScript`).